src/BUILD.md

```diff
@@ -4,7 +4,10 @@ Running the X-Road software requires Linux (Ubuntu or RHEL). As a development en
 
 **Tools**
 
-*Required for building*
+*Required for building in Docker (recommended)*
```

Comment:
I have weaved in some text into the introduction. Actually building in docker is the only option for me because I cannot afford "taining" my machine with stuff not related to my work. But we know from experience that longer a README gets the less people will read it :)